tests/common.go

```diff
@@ -0,0 +1,226 @@
+package tests
+
+import (
+	"flag"
```

Comment:
did we miss out on the log bundle change?

Reply:
Adding it back in incremental.